### PR TITLE
Cookie validation common

### PR DESCRIPTION
**Pull request by Faryshta**
| Q             | A
| ------------- | ---
| Is bugfix?    | no (test fix)
| New feature?  | no
| Breaks BC?    | no
| Tests pass?   | yes
| Fixed issues  | #346

### CODE REVIEW

**Comment by samdark** [comment]
@Faryshta won't it make console application to error because there's no `cookieValidationKey` in its request component?

**Comment by Faryshta** [comment]
@samdark just verified and yes it does cause an error. Will provide another solution.

**Comment by Faryshta** [comment]
@samdark `yii_test` and `codecept run` have different configuration files now on backend, common and frontend. Both commands work properly.

File: environments/dev/common/config/codeception-local.php [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<?php
+
+return yii\helpers\ArrayHelper::merge(
+    require __DIR__ . '/main.php',
+    require __DIR__ . '/main-local.php',
+    require __DIR__ . '/test.php',
```
Comment (Faryshta) [inline environments/dev/common/config/codeception-local.php]: I dont know if I should add '/test-local.php' here aswell

Comment (Faryshta) [reply environments/dev/common/config/codeception-local.php]: @samdark done

**Comment by samdark** [comment]
Merged. Thank you!

**Pull request merged**